fuel-moshi/src/main/kotlin/com/github/kittinunf/fuel/moshi/FuelMoshi.kt

```diff
@@ -20,3 +20,8 @@ inline fun <reified T : Any> moshiDeserializerOf() = object : ResponseDeserializ
                     .adapter(T::class.java)
                     .fromJson(content)
 }
+
+inline fun <reified T : Any> moshiDeserializerOf(adapter: JsonAdapter<T>) = object : ResponseDeserializable<T> {
```

Comment:
My concern about inline fun that Kotlin added $inline$ classes for it if you look into the Kotlin Bytecode Viewer.

Reply:
Hm okay, but how is this a problem? It should also be the case when using the functions above it, right?

Reply:
When I removed inline from the  moshiDeserializerOf, the Test Coverage able to hit it.

Reply:
But we should rather file an issue @ Codecov (or whatever you use) that adjust the API because of glitchy test coverage.